recipes/perl/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set base_version = "5.20.3" %}
+{% set version = base_version + ".1" %}
+
+
+package:
+  name: perl
+  version: {{ version }}
+
+source:
+  fn: perl-{{ base_version }}.tar.gz  #[unix]
+  fn:  strawberry-perl-{{ version }}-32bit-portable.zip  #[win]
```

Comment:
It was a version of Perl that @msarahan found that is built for Windows. Will hold us over until we can get a proper build system for Windows within conda.